test/runner/lib/sanity/ansible_doc.py

```diff
@@ -67,19 +75,19 @@ def test(self, args, targets, python_version):
         target_paths = collections.defaultdict(dict)
 
         for module in modules:
-            doc_targets['module'].append(module)
+            doc_targets['module'].append(data_context().content.prefix + module)
```

Comment:
should this be `os.path.join` instead?

Reply:
No, since `data_context().content.prefix` is not a path.

When testing Ansible it is `""` and when testing a collection it is in the form `"{namespace}.{collection}."` so that collection names passed to ansible-doc are fully qualified.